src/components/common/ui/form/input-mui/Input.styles.ts

```diff
@@ -0,0 +1,164 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+import {
+  InputSize,
+  InputState,
+  InputType,
+} from '@/components/common/ui/form/input-mui/types';
+import theme from '@/styles/theme';
+
+export const wrapper: SxProps<Theme> = {
+  width: '100%',
+  padding: 0,
+};
+
+export const label = (state: InputState): SxProps<Theme> => ({
+  overflow: 'hidden',
+  padding: '2px 8px',
+  maxWidth: '78%',
+  background: `linear-gradient(180deg, rgba(30, 30, 30, 0) 50%, ${theme.palette.backgroundDark[50]} 49.95%)`,
+
+  ...(state === InputState.ERROR && {
+    color: theme.palette.error[500],
+    '&.Mui-focused': {
+      color: theme.palette.error[500],
+    },
+  }),
+
+  ...(state === InputState.SUCCESS && {
+    color: theme.palette.success[600],
+    '&.Mui-focused': {
+      color: theme.palette.success[600],
+    },
+  }),
+
+  ...(state === InputState.DEFAULT && {
+    color: theme.palette.grey[800],
+    '&.Mui-focused': {
+      color: theme.palette.grey[800],
+    },
+  }),
+
+  '&.Mui-disabled': {
+    color: theme.palette.grey[400],
+  },
+});
+
+export const remark = (state: InputState): SxProps<Theme> => ({
+  margin: '2px 8px 0 16px',
+  '&.MuiFormHelperText-root': {
+    textTransform: 'lowercase',
+  },
+
+  ...(state === InputState.ERROR && {
+    color: theme.palette.error[500],
+  }),
+
+  ...(state === InputState.SUCCESS && {
+    color: theme.palette.success[600],
+  }),
+
+  typography: 'overline',
+  minHeight: '20px',
+});
+
+export const input = (state: InputState, size: InputSize): SxProps<Theme> => ({
+  transition: 'all 0.2s ease-in-out',
+  width: '100%',
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  gap: '8px',
+  backgroundColor: 'backgroundDark.50',
+  borderRadius: '8px',
+  resize: 'none',
+  border: '2px solid',
+  color: theme.palette.grey[800],
+
+  ...(size === InputSize.LARGE && {
+    padding: '14px 16px',
+    typography: 'body2',
+  }),
+
+  ...(size === InputSize.MEDIUM && {
+    padding: '13px 12px',
+    typography: 'body1',
+  }),
+
+  ...(state === 'default' && {
```

Comment:
you are using passing enum type so use InputState instead string as it was in `remark`
Same for error, success